gitops/fleet/charts/fleet-argo/values-test.yaml

```diff
@@ -0,0 +1,90 @@
+name: myteams
```

Comment:
What are those teams ? We deploy application for the web store website ?

Reply:
That will be removed as well on the v2 gitops-bridge it was just for testing puerposes